physics/src/IceGrowth.cpp

```diff
@@ -154,6 +160,10 @@ void IceGrowth::update(const TimestepTime& tsTime)
                          std::placeholders::_2),
             tsTime);
     }
+
+    // Damage always heals, even if there's no active thermo
+    // TODO: This should only be called for brittle rheologies
+    iHealing->update(tsTime);
```

Comment:
IDynamics provides a `usesDamage()` function. Calling this on the currently active dynamics implementation would allow the healing function to be called only on brittle (damage-using) rheologies.